README.md

```diff
@@ -97,10 +96,10 @@ Generally creating a level looks like:
 SVG File -> Custom Level File -> Game
 ```
 
-To convert SVG to the level file and run [svg2rects.py] script:
+To convert SVG to the level file and run svg2level program:
 
 ```console
-$ python3 ./devtools/svg2rects.py -i <svg-file> -o <level-file>
+$ ./build/svg2level <svg-file> <level-file>
```

Comment:
i was talking about this: shouldn't it be `build/svg2level compile <svg-file> <level-file>` ?

Reply:
@zhiayang you're right! Thanks!